### PR TITLE
Frontend testing

### PR DESCRIPTION
**Pull request by jricks86**
Testing for Header, Footer, Notfound, and Home Passing

### CODE REVIEW

File: src/Pages/Home.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 import React from "react"
 
 const Home = () => {
-    
+    return (
+        <>
+        <h2>Home Page</h2>
```
Comment (SunkissedQueen) [inline src/Pages/Home.js:5]: Be mindful of indentation

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Just a quick fix on the indentation. Looks like you are comfortable testing to see if the component does not crash. I recommend that you try some queries. Use the playground tool that the react testing library provides.

**Comment by jricks86** [comment]
Added more testing

**Review by SunkissedQueen — APPROVED** [review]
As of right now all the tests are verifying that the components are rendering. When you are complete with your project, please got back and review the query that can be made in the react testing library and check out the playground debugging tool.

**Pull request merged**